openedx/adg/admin_dashboard/admin_settings/views.py

```diff
@@ -278,6 +353,30 @@ def get_queryset(self):
 
         return invitations
 
+    def get_status(self, action_type, email):
+        """
+        Return status based on the action the user performed and the saved invitation against an email if there
+        exists any otherwise return an error.
+
+        Arguments:
+            action_type (str): indicates the type of action the user performed
+            email (str): email of the admin being invited/updated/removed
+
+        Returns:
+            int: invitation status
+            Invitation: Invitation object
+            str: error
+        """
+        if action_type == INVITATION:
+            return Invitation.PENDING, None, ''
+        elif action_type == EDIT_INVITATION:
+            invitation_obj = Invitation.objects.get_invitation_for_email(email)
+
+            if invitation_obj:
+                return Invitation.PENDING, invitation_obj, ''
+            else:
+                return '', None, NO_INVITATION_ERROR.format(email=email)
```

Comment:
Doc string says it will return `int` but  here its returning empty string. Is it intentional?